charts/civil-service/values.yaml

```diff
@@ -46,11 +46,11 @@ java:
     CMC_DB_NAME: cmc
     CMC_DB_PORT: 5432
     REFERENCE_DATABASE_MIGRATION: true
-    SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_ISSUERURI: https://forgerock-am.service.core-compute-idam-{{ .Values.global.environment }}.internal:8443/openam/oauth2/hmcts
+    SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_ISSUERURI: https://idam-api.{{ .Values.global.environment }}.platform.hmcts.net/openam/oauth2/hmcts
     SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_JWKSETURI: https://idam-api.{{ .Values.global.environment }}.platform.hmcts.net/o/jwks
     AUTH_IDAM_CLIENT_BASEURL: https://idam-api.{{ .Values.global.environment }}.platform.hmcts.net
     AUTH_PROVIDER_SERVICE_CLIENT_BASEURL: http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
-    OIDC_ISSUER: https://forgerock-am.service.core-compute-idam-{{ .Values.global.environment }}.internal:8443/openam/oauth2/realms/root/realms/hmcts
+    OIDC_ISSUER: https://idam-api.{{ .Values.global.environment }}.platform.hmcts.net/openam/oauth2/realms/root/realms/hmcts
```

Comment:
The ExUI team are not sure if these values for Issuer will work. Looking at the IDAM docs https://tools.hmcts.net/confluence/display/SISM/OIDC+Integration+Guide 
Then the issue URL should be something like 
https://idam-web-public.{env}.platform.hmtcts.net/o